data-tiles/cmd/geolookup/testdata/out.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "en": "Hartlepool",
+    "geoType": "LAD",
+    "geoCode": "E06000001",
+    "bbox": [
```

Comment:
its not going to break anything to have the bbox here, but its not being used any more by the front end, and it must add quite a bit to the filesize. Would it be a pain to remove?